cpp/JsiWorkletContext.h

```diff
@@ -144,7 +144,7 @@ class JsiWorkletContext
    Returns the list of decorators
    */
   static const std::vector<std::shared_ptr<JsiBaseDecorator>> &getDecorators() {
-    return decorators;
+    return std::vector<std::shared_ptr<JsiBaseDecorator>>();
```

Comment:
Returns a pointer to a temporary object

```suggestion
  return std::move(std::vector<std::shared_ptr<JsiBaseDecorator>>());
```

Also this method is never used - do we need it?

Reply:
Ah yeah whoops that should've been removed